src/Logikfabrik.Umbraco.Jet/Mappings/DataTypeDefinition.cs

```diff
@@ -25,6 +25,11 @@ public enum DataTypeDefinition
         ContentPicker = 1034,
 
         /// <summary>
+        /// Data type for new content picker.
+        /// </summary>
+        ContentPicker2 = 1046,
```

Comment:
Good point there, actually Umbraco just added "2" to the new editors as well. I've picked them up from the cmsDataType table:

pk | nodeId | propertyEditorAlias | dbType
--- | --- | --- | ---
27 | 1047 | Umbraco.MemberPicker2 | Nvarchar
28 | 1048 | Umbraco.MediaPicker2 | Ntext
29 | 1049 | Umbraco.MediaPicker2 | Ntext
30 | 1050 | Umbraco.RelatedLinks2 | Ntext